hw/drivers/sensors/lis2dw12/src/lis2dw12.c

```diff
@@ -1909,14 +1909,11 @@ int lis2dw12_run_self_test(struct sensor_itf *itf, int *result)
 
     /* calculate accel data difference */
     change = 0;
+    *result = 0;
     for(i = 0; i < 3; i++) {
-        change += pos[i] - base[i];
-    }
-
-    if ((change > 70) && (change < 1500)) {
-        *result = 0;
-    } else {
-        *result = -1;
+    	change = pos[i] - base[i];
+    	if(!(change > 70) && (change < 1500))
```

Comment:
Coding style: need whitespace after 'if'. More importantly: even single line statements after if-conditions must be wrapped with { }.